delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/resources/Resources.scala

```diff
@@ -90,6 +90,22 @@ trait Resources {
       source: Json
   )(implicit caller: Caller): IO[DataResource]
 
+  /**
+    * Update the schema of an existing resource
+    *
+    * @param id
+    *   identifier that will be expanded to the iri of the resource
+    * @param projectRef
+    *   project reference where the resource belongs
+    * @param schema
+    *   identifier of the new schema that will be used to validate the resource. This identifier will be expanded
+    */
+  def updateResourceSchema(
```

Comment:
`updateAttachedSchema` ?

Reply:
Is this a suggestion just for this method or also for the event/command name?

Reply:
Just for this method